source/Ship.cpp

```diff
@@ -2211,6 +2211,9 @@ void Ship::DoGeneration()
 			for(const pair<double, Ship *> &it : carried)
 			{
 				Ship &ship = *it.second;
+				if(ship.hull < ship.MinimumHull() || ship.isDisabled)
```

Comment:
Do we need to check both of these? I think that isDisabled checks the hull vs mininum hull too?